vendor/parser-extended/parser.mly

```diff
@@ -1375,19 +1373,40 @@ structure_item:
     { $1 }
 ;
 
+%inline ext_attrs(kw, body):
+  kw
+  ext = ext 
+  before = attributes
+  body = body
+  after = post_item_attributes
+  { let loc = make_loc $sloc in
+    let docs = symbol_docs $sloc in
+    let attrs = Attr.ext_attrs ?ext ~before ~after () in 
+    body ~loc ~attrs ~docs ~text:None }
+
+
+%inline ext_attrs_no_ext(kw, body):
+  kw
+  before = attributes
+  body = body
+  after = post_item_attributes
+  { let loc = make_loc $sloc in
+    let docs = symbol_docs $sloc in
+     let text = Some (symbol_text $symbolstartpos) in
+    let attrs = Attr.ext_attrs ~before ~after () in 
+    body ~loc ~attrs ~docs ~text }
+
 (* A single module binding. *)
 %inline module_binding:
-  MODULE
-  ext = ext attrs1 = attributes
-  name = mkrhs(module_name)
-  args = functor_args
-  body = module_binding_body
-  attrs2 = post_item_attributes
-    { let docs = symbol_docs $sloc in
-      let loc = make_loc $sloc in
-      let attrs = attrs1 @ attrs2 in
-      let body = Mb.mk name args body ~attrs ~loc ~docs in
-      Pstr_module body, ext }
+  ext_attrs (
+    MODULE,
```

Comment:
The ext_attrs macro is nice but the large diffs to the parser are problematic. 
Upstream changes more and more often and I don't want backporting to be slowed down.

I'd prefer to keep the rule as before and to change only the `let attrs` line in the body.

Reply:
This is still here. Did you forget to check something in your last commit ?